CONTRIBUTING.md

```diff
@@ -31,13 +35,13 @@ For small documentation changes and fixes, these can be done quickly following t
 
 ### Cloning (or forking) the repository
 
-First off, you are going to need your own copy of the repository. You can find help on how to fork a repo [here](https://help.github.com/articles/fork-a-repo/).
+First off, find help on how to fork a repo [here](https://help.github.com/articles/fork-a-repo/).
```

Comment:
The heading here sounds like we are being inclusive of `cloning`, but I'd assume we want contributors to `fork` - almost always. Unless I'm mistaken, maybe we can remove the "Cloning" part of the heading to just leave "Forking the repository"

Reply:
yea definitely want to encourage forking to then make PR's back. cloning doesn't really promote that